### PR TITLE
Send email to all expired orders

### PR DESCRIPTION
**Pull request by aditya1702**
#1959

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -350,17 +370,38 @@ def send_after_event_mail():
...
+            send_email_for_expired_orders('adityavyas17@gmail.com', order.event.name, order.get_invoice_number(),
+                                          url_for('ticketing.view_order_after_payment',
+                                                  order_identifier=order.identifier, _external=True))
 
 
+#logging.basicConfig()
+order_sched = BackgroundScheduler(timezone=utc)
+order_sched.add_job(send_mail_to_expired_orders,
+                    'interval',
+                    seconds=10)
```
Comment (niranjan94) [inline app/__init__.py]: Every 10 seconds is too less of an interval for this job ... make it 30 mins or atleast an hour ... We don't want the user to get an email as soon as he leaves the site ... If he gets an email after a while, there's more chance of him/her rethinking and changing their mind

**Comment by aditya1702** [comment]
@niranjan94 Yes I know. I am still testing it thats y I kept the time less. In the final changes I will make the time more :)

**Comment by niranjan94** [comment]
@aditya1702 Oh ... Okay ! ... :D .. sorry

**Comment by aditya1702** [comment]
@niranjan94 Please review

**Pull request merged**